.travis.yml

```diff
@@ -56,6 +56,13 @@ before_cache:
   - brew cleanup
 
 before_install:
+  - docker pull bitnami/zookeeper:3.6.0
```

Comment:
We should install dependencies in specific scripts, eg: https://github.com/apache/incubator-apisix/blob/master/.travis/linux_openresty_runner.sh#L35

It may be a little between different os, eg: Linux and OSX.